tests/test_objects.py

```diff
@@ -719,6 +732,7 @@ def test_make_pod_with_extra_pod_config():
                         "containerPort": 8888
                     }],
                     'volumeMounts': [],
+                    'restartPolicy': 'OnFailure',
```

Comment:
Restart policy in extra_pod_config was set to Always, but should output OnFailure? Probably not right?

I was looking to find why you had a build failure, this isn't the main reason but it will probably need to be fixed as well.

Reply:
Oh hmmm wait, this is the containers restartPolicy, eehmmm okay I need to read up on Pod / Container restart-policies.